ui/page/components/account_selector.go

```
@@ -234,16 +241,17 @@ func newAccountSelectorModal(l *load.Load, currentSelectedAccount *dcrlibwallet.
 }
 
 func (asm *AccountSelectorModal) OnResume() {
-	wallets := make([]*dcrlibwallet.Wallet, 0)
-	walletAccounts := make(map[int][]*selectorAccount)
+	asm.ctx, asm.ctxCancel = context.WithCancel(context.TODO())
+	asm.setupAccountBalance()
+	asm.listenForTxNotifications()
+}
 
-	// TODO use a sorted wallet list
-	for _, wal := range asm.wallets {
-		// filter all accounts
+func (asm *AccountSelectorModal) setupAccountBalance() {
+	walletAccounts := make(map[int][]*selectorAccount)
+	for _, wal := range asm.WL.SortedWalletList() {
 		accountsResult, err := wal.GetAccountsRaw()
 		if err != nil {
-			fmt.Println("Error getting accounts:", err)
-			return
+			continue
```

Why completely ignoring this error? At least a log is necessary.